documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -184,6 +184,20 @@ endif::[]
 |
 | Set this parameter to an available IP address on the `baremetal` network when the `provisioningNetwork` configuration setting is set to `Disabled`.
 
+ifeval::[{release} > 4.4]
+| `httpProxy`
+|
+| Set this parameter to the appropriate http_proxy used within your environment.
+
+| `httpsProxy`
+|
+| Set this parameter to the appropriate https_proxy used within your environment.
+
+| `noProxy`
+|
+| Set this parameter to the appropriate no_proxy used within your environment.
```

Comment:
appropiate list of exclussions for proxy usage within your environment? (aligned with prior two comments), disregards them as a whole